src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -33,6 +33,10 @@ class YarnBuilder extends AbstractBuilder {
     yarn("sonar-scan ${properties}")
   }
 
+  def highLevelDataSetup(String dataSetupEnvironment) {
+    yarn("highLevelDataSetup --dataSetupEnvironment=${dataSetupEnvironment}")
```

Comment:
maybe this would be easier? not a blocker
```suggestion
    yarn("highLevelDataSetup ${dataSetupEnvironment}")
```

Reply:
Resolved in [fbfecd4acaa5625b8f1f9047885f2c84fa555ced](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/fbfecd4acaa5625b8f1f9047885f2c84fa555ced)